lib/addons.js

```diff
@@ -49,7 +49,7 @@ module.exports = function (app, log, client) {
       then(
         function (addonInfo) {
           if (addonInfo.plan.name === addonConfig.plan) {
-            if (addonInfo.name !== addonConfig.name) {
+            if (addonInfo.name !== addonConfig.name && addonConfig.name !== undefined) {
```

Comment:
Can we do if (addonConfig.name && addonInfo.name) ?
It will cover null, undefined and empty strings. We had a bag in the past where null was breaking some part of a code where only undefined was being checked with ===.